pkg/dom0-ztools/rootfs/bin/eve

```diff
@@ -100,7 +100,7 @@ http_debug_request() {
     fi
 
     if [ "$running" = "0" ]; then
-        pkill -USR2 zedbox
+        pkill -o -USR2 zedbox
```

Comment:
Why would there be more than one zedbox process?

Reply:
When zedbox tries to execute f.e. `qemu-img`, then zedbox first does a `fork()` and then and `exec*`. In between there are two, aren't there?

Reply:
I don't know how the golang does the Exec - it has some constraints thus it might (need to) block signals across some of this in the golang runtime. FWIW I know have three observed crashes which indicates that something changed in master over the last two days causing the crashes.

Reply:
```
package main

import (
	"fmt"
	"os/exec"
)

func main() {
	cmd := exec.Cmd{
		Path: "/bin/ls",
	}

	out, err := cmd.Output()
	if err != nil {
		panic(err)
	}

	fmt.Printf("%s\n", out)
	cmd.Run()

	cmd.Wait()

}
```

[exec.strace.txt](https://github.com/user-attachments/files/15985680/exec.strace.txt)

child process is `216343`

in line 344 I see:
```
216337 clone(child_stack=NULL, flags=CLONE_VM|CLONE_VFORK|SIGCHLD <unfinished ...>
```

and soon after (line 370):
```
216343 rt_sigaction(SIGUSR2, {sa_handler=SIG_DFL, sa_mask=~[], sa_flags=SA_RESTORER|SA_ONSTACK|SA_RESTART|SA_SIGINFO, sa_restorer=0x469ce0}, NULL, 8) = 0
```
the signal handler for USR2 gets removed.

in line 472 the exec syscall is finally invoked.